### ISSUE TITLE
Move the flatpak test to the "Install packages" section in travis

### ISSUE THREAD

**Issue by madoar** [PhoenicisOrg/phoenicis#1494]
Currently the flatpak test is run in the section "Build + test".
I think it would be more fitting to move the test to the "Install packages" section.

**Comment by plata** [comment]
It's not installed. It's only built. Also, in the build+test stage, we can test in parallel.

**Comment by madoar** [comment]
My problem with the section choice is, that under "Build + test" I understand the commands `mvn clean compile` and `mvn clean test`.
While everything we do afterwards with the built jar should be part of another section.

**Comment by plata** [comment]
"Build" is not specific. Building a flatpak is also "build". I can understand your point nevertheless. However, the .deb and .dmg are also built in the "Build + test" stage.

**Comment by madoar** [comment]
Yes, but it's part of the normal java build.
So maybe let me replace `mvn clean compile` with `mvn clean package`.
If we can add the flatpak creation to the maven lifecycle I would be fine with it.

Another issue is, that we're mainly testing different java versions in the `Build + test` section.
Here the flatpak test doesn't really fit in.

**Comment by madoar** [comment]
The plugin info package seems to contain only minimal information.
https://www.eclipse.org/cbi/sitedocs/eclipse-flatpak-packager/package-flatpak-mojo.html contains a list of plugin parameters.

**Comment by plata** [comment]
I've seen that. But it's still not clear for me. Also, I cannot find anything about this on the Internet. That's never a good indicator.
